tests/specs/mcd/verification.k

```diff
@@ -326,13 +351,18 @@ module LEMMAS-MCD-COMMON
 
     rule M:Memory [ K1 := (BUF11 ++ BUF12) ] [ K2 := BUF2 ] => M [ K1 +Int #sizeByteArray(BUF11) := BUF12 ] [ K2 := BUF2 ]
       requires K2 <=Int K1
-       andBool K1 +Int #sizeByteArray(BUF11) <=Int K2 +Int #sizeByteArray(BUF2)
+      andBool K1 +Int #sizeByteArray(BUF11) <=Int K2 +Int #sizeByteArray(BUF2)
```

Comment:
```suggestion
       andBool K1 +Int #sizeByteArray(BUF11) <=Int K2 +Int #sizeByteArray(BUF2)
```

I like the `l` at the end of `andBool` to line up with the `s` at the end of `requires`, so that all the conditions are horizontally aligned.